pkg/security/probe/probe_windows.go

```diff
@@ -91,15 +91,15 @@ func (p *WindowsProbe) Start() error {
 					continue
 				}
 
-				log.Tracef("Received start %v", start)
+				log.Debugf("Received start %v", start)
 
-				ppid, err := procutil.GetParentPid(pid)
-				if err != nil {
-					log.Errorf("unable to resolve parent pid %v", err)
-					continue
-				}
+				// TODO
+				// handle new fields
+				// CreatingPRocessId
+				// CreatingThreadId
+				// OwnerSidString
 
-				pce, err = p.Resolvers.ProcessResolver.AddNewEntry(pid, ppid, start.ImageFile, start.CmdLine)
+				pce, err = p.Resolvers.ProcessResolver.AddNewEntry(pid, uint32(start.PPid), start.ImageFile, start.CmdLine)
```

Comment:
From what I can tell, `PPid` is a `unit64`, why the cast to `uint32`?

Reply:
The process resolver apparently takes 32 bit ints.
PIDS are defined by MS as DWORDs (32 bit), so there's no loss of fidelity.
I generally have the fields coming up from the driver as 64 bit for byte alignment purposes.  So it's just wasting the upper 32 bits.